### PR TITLE
Fix consistent hashing

### PR DESCRIPTION
**Pull request by CoderKevinZhang**
**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: globalscheduler/controllers/util/consistenthashing/consistent_hashing.go [hunk 1]
```diff
@@ -184,14 +184,11 @@ func (ch *ConsistentHash) Insert(names []string) error {
...
-			// find the first index of hash value that is greater than key
-			idx := ch.search(key)
-			ch.Members[v] = ch.HashCircle[ch.SortedHashes[idx]]
-			ch.Results[ch.HashCircle[ch.SortedHashes[idx]]] = append(ch.Results[ch.HashCircle[ch.SortedHashes[idx]]], v)
-		}
+		key := ch.fnv32Hash(v)
+		// find the first index of hash value that is greater than key
+		idx := ch.search(key)
+		ch.Members[v] = ch.HashCircle[ch.SortedHashes[idx]]
+		ch.Results[ch.HashCircle[ch.SortedHashes[idx]]] = append(ch.Results[ch.HashCircle[ch.SortedHashes[idx]]], v)
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/util/consistenthashing/consistent_hashing.go:191]: LGTM

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Review by kimeunju108 — APPROVED** [review]
I approved the PR , 
But I am wondering why it assigned sequentially (e.g. 0, 1 2 3 4 5 6 7 8 9). It should be random. 
map[node-1:[0 1 2 3 4 5 6 7 8 9] node-3:[10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29]]

**Review by jshaofuturewei — APPROVED** [review]
LGTM

**Pull request merged**